docs/install.md

```diff
@@ -1,26 +1,167 @@
+
+1. `K0S_VERSION=v0.11.0` which allows you to pick k0s version v0.11.0
+2. `DEBUG=true` Prints commands and their arguments as they are executed.
+
+### cURL
+
+This example is essentially what is described in section [above](#shellscript) with the difference that the binary that is downloaded will keep the name as it is and that `chmod` needs to be set to `755` or `+x`
+```
+curl -LJO  "https://github.com/k0sproject/k0s/releases/download/v0.11.0/k0sctl-linux-x64"
+```
+
+# Installing k0s as a service on the local system
+
+The `install` sub-command will install k0s as a system service on hosts running one of the supported init systems (systemd and OpenRC).
+
+Install can be executed for two k0s roles: controller and worker.
+
+Install `controller` subcommand will accept all available flags and parameters supported.
+
+```
+$ k0s install controller --help
+
+Helper command for setting up k0s as controller node on a brand-new system. Must be run as root (or with sudo)
+
+Usage:
+  k0s install controller [flags]
+
+Aliases:
+  controller, server
+
+Examples:
+All default values of controller command will be passed to the service stub unless overriden. 
+
+With controller subcommand you can setup a single node cluster by running:
+
+        k0s install controller --enable-worker
+
+
+Flags:
+  -c, --config string            config file (default: ./k0s.yaml)
+      --cri-socket string        contrainer runtime socket to use, default to internal containerd. Format: [remote|docker]:[path-to-socket]
+  -d, --debug                    Debug logging (default: false)
+      --enable-worker            enable worker (default false)
+  -h, --help                     help for controller
+  -l, --logging stringToString   Logging Levels for the different components (default [konnectivity-server=1,kube-apiserver=1,kube-controller-manager=1,kube-scheduler=1,kubelet=1,kube-proxy=1,etcd=info,containerd=info])
+      --profile string           worker profile to use on the node (default "default")
+      --token-file string        Path to the file containing join-token.
+
+Global Flags:
+      --data-dir string        Data Directory for k0s (default: /var/lib/k0s). DO NOT CHANGE for an existing setup, things will break!
+      --debugListenOn string   Http listenOn for debug pprof handler (default ":6060")
+```
+
+For example running a single node k0s cluster on Ubuntu 20.10:
+
+```
+$ k0s install controller --enable-worker
+INFO[2021-02-24 11:05:42] no config file given, using defaults         
+INFO[2021-02-24 11:05:42] creating user: etcd                          
+INFO[2021-02-24 11:05:42] creating user: kube-apiserver                
+INFO[2021-02-24 11:05:42] creating user: konnectivity-server           
+INFO[2021-02-24 11:05:42] creating user: kube-scheduler                
+INFO[2021-02-24 11:05:42] Installing k0s service
+```
+
+Run k0s as a service
+
+```
+$ systemctl start k0scontroller.service
+```
+
+Check service status:
+
+```
+# systemctl status
```

Comment:
```suggestion
# systemctl status k0scontroller.service
```

Reply:
Is this command supposed to be the output of systemctl, or k0s status command? (it's not clear)

Reply:
You don't need the `.service`.